pinc/ProjectTransition.inc

```diff
@@ -1257,7 +1338,7 @@ new ProjectTransition(
         'action_name'           => '[default]',
         'confirmation_question' => $post_complete_question,
         'detour'                => null,
-        'settings_template'     => '',
+        'settings_template'     => "ppverifier=NULL",
```

Comment:
As the `$post_complete_question` says, currently using this transition probably isn't a good idea, since we don't really have a workflow for it. However, if it is used, and if there was a PPVer involved, we should consider whether it's appropriate to include them? (edit: I don't know the answer off the top of my head -- just raising the question.)